tests/gold_tests/pluginTest/cert_update/cert_update.test.py

```diff
@@ -33,25 +35,22 @@
 server.addResponse("sessionlog.json", request_header, response_header)
 
 # Set up ATS
-ts = Test.MakeATSProcess("ts", command="traffic_manager", select_ports=False)
+ts = Test.MakeATSProcess("ts", command="traffic_manager", enable_tls=1)
 
 # Set up ssl files
 ts.addSSLfile("ssl/server1.pem")
 ts.addSSLfile("ssl/server2.pem")
 ts.addSSLfile("ssl/client1.pem")
 ts.addSSLfile("ssl/client2.pem")
 
-# Setup ssl ports
-ts.Variables.ssl_port = 4443
-s_server_port = 12345
+# reserve port, attach it to 'ts' so it is released later
+ports.get_port(ts, 's_server_port')
```

Comment:
Why don't you just use the auto port selection when calling Test.MakeOriginServer?

Reply:
Oh, nevermind, I see you are using this port for "openssl s_server".  Looks good.  Should document this someplace.

Reply:
this is the one case i found that should probably use @ywkaras #6291 because the port isn't really used by TS, be it will have the same lifetime so its fine like this.